packages/lib/modules/tokens/approvals/permit/signRemoveLiquidityPermit.tsx

```diff
@@ -50,15 +50,22 @@ async function signPermit({ permitInput, wethIsEth, sdkClient, pool }: Params):
     sdkQueryOutput: permitInput.sdkQueryOutput as RemoveLiquidityQueryOutput,
   })
 
-  const signPermitFn = isBoosted(pool)
-    ? PermitHelper.signRemoveLiquidityBoostedApproval
-    : PermitHelper.signRemoveLiquidityApproval
-
-  const signature = await signPermitFn({
+  const baseParams = {
     ...baseInput,
     /* eslint-disable @typescript-eslint/no-non-null-assertion */
     client: sdkClient!,
     owner: permitInput.account,
-  })
-  return signature
+  }
+
+  if (isV3WithNestedActionsPool(pool)) {
+    return PermitHelper.signRemoveLiquidityNestedApproval({
+      ...baseParams,
+      // TODO: We can inline baseParams if the SDK renames bptAmountIn to bptIn to match the naming with the rest of the output types
+      // bptAmountIn: baseInput.bptIn,
```

Comment:
This is causing issues. 
An SDK review is needed cause other sign methods have `bptIn` instead of `bptAmountIn`